pkg/kubelet/network/plugins.go

```diff
@@ -54,6 +60,9 @@ type NetworkPlugin interface {
 	// for a plugin by name, e.g.
 	Name() string
 
+	// Returns a bitfield of plugin capabilities
+	Capabilities() uint
```

Comment:
> nit: I prefer returning a Set instead of uint.

Do you mean an Int Set or a String Set?  I only get one hit for Int sets (pkg/client/restclient/urlbackoff.go) so it seems like there aren't a ton of Int/Int64/Byte Set users yet.  I'm OK moving to a Set though if you want one.